client/lib/state-persistence/worker.js

```diff
@@ -0,0 +1,83 @@
+const getWorker = ( () => {
+	let worker;
+
+	function createWorker() {
+		const blob = new Blob( [ `
+			importScripts( 'https://cdnjs.cloudflare.com/ajax/libs/localforage/1.4.3/localforage.js' );
```

Comment:
If we don't end up using `lib/localforage`, let's at least use a named db. The default will use localforage:
<img width="1011" alt="screen shot 2016-12-12 at 12 50 05 pm" src="https://cloud.githubusercontent.com/assets/1270189/21117043/09d6e204-c06d-11e6-8adb-8e11452f6711.png">